medusa/server/web/config/post_processing.py

```diff
@@ -213,19 +220,13 @@ def isNamingValid(pattern=None, multi=None, abd=False, sports=False, anime_type=
             return 'invalid'
 
     @staticmethod
-    def isRarSupported():
-        """
-        Test Packing Support:
-            - Simulating in memory rar extraction on test.rar file
-        """
-
+    def isRarSupported(unrar_tool):
```

Comment:
So you want to expand a PR that replaces an old RAR dependency with a new one, to a PR that also replaces about 166 routes? Those are not just doing search/replaces. But also all Mako templates, js files need to be changed and tested. And of course updating the documentation, for the new convention. And I bet i'm not overseeing all changes. So we'll probably get some unexpected errors.